### PR TITLE
cleanup imports; remove pre-push linting

### PR DESCRIPTION
**Pull request by timmyichen**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Chapter.

The PR:
* Cleans up factory imports to use aliases instead of `../`
* Removes husky pre-push linting.  It's pretty slow (takes a few minutes to run on my machine) and there's no need to do this prior to pushing.  Any linting errors will be caught in CI; or, if you're unsure, it can be run itself.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -131,4 +130,4 @@
     "node": "10.x",
     "npm": ">= 6.0.0"
   }
-}
+}
```
Comment (Zeko369) [inline package.json]: Why did you remove the newline?

Comment (timmyichen) [reply package.json]: Accident, fixing!
